src/utils/sheetID.ts

```diff
@@ -0,0 +1,3 @@
+export const EVENT_SHEET_ID = "EVENTS";
```

Comment:
I noticed that there are a bunch of files in the utils folder. First of all, our file naming scheme is snake_case. 

Next, don't have these be in their own file but instead put them in something applicable like `sheets.ts`

Reply:
So, please rename all the files to their snake_case equivalent.